docs/user_guide.rst

```diff
@@ -259,6 +259,49 @@ optimizations. Let us try it out on our decomposed and routed circuit above:
     gates for a particular device. In the example above we don't have them, however it is generally a good idea to run
     decomposition once again after the simplification.
 
+Classical control
+-----------------
+Some IQM quantum computers support classically-controlled gates, that is, gates that are executed
+conditionally depending on the result of a measurement preceding them in the quantum circuit. This
+support currently has several limitations:
+
+* Only the ``x``, ``y``, ``rx``, ``ry`` and ``r`` gates can be classically controlled.
+* The gates can only be conditioned on one classical bit, and the only control available is to
+  apply the gate if the bit is 1, and apply an identity gate if the bit is 0.
+* The availability of the controlled gates depends on the instrumentation of the quantum computer.
+
+The classical control can be applied on a circuit operation using :meth:`~cirq.Operation.with_classical_controls`:
```

Comment:
```suggestion
The classical control can be applied on an operation using :meth:`~cirq.Operation.with_classical_controls`:
```
In Cirq CircuitOperation is a different class, so to avoid confusion